### ISSUE TITLE
"Adapters For Interoperability between PSR-6 and PSR-16 Cache" wrong class name?

### ISSUE THREAD

**Issue by peaceant** [symfony/symfony-docs#11824]
https://github.com/symfony/symfony-docs/blob/master/components/cache/psr6_psr16_adapters.rst

In the last code example is new instance of `Psr6Cache`. Shouldn't it be instance of `Psr16Cache` since you are importing `Symfony\Component\Cache\Psr16Cache`?

```php
use Symfony\Component\Cache\Adapter\FilesystemAdapter;
use Symfony\Component\Cache\Psr16Cache;

// the PSR-6 cache object that you want to use
$psr6Cache = new FilesystemAdapter();

// a PSR-16 cache that uses your cache internally!
$psr16Cache = new Psr6Cache($psr6Cache); # <--- Wrong?
$psr16Cache = new Psr16Cache($psr6Cache); # <--- Correct?

// now use this wherever you want
$githubApiClient = new GitHubApiClient($psr16Cache);
```

**Comment by nicolas-grekas** [comment]
Correct, PR welcome.

**Comment by javiereguiluz** [comment]
Closing as fixed by #11830.
